src/components/UserInputBlock.tsx

```diff
@@ -48,23 +48,21 @@ export const UserInputBlock = ({ content, prompt, ack, waitForEvent, fireEvent,
         <div ref={contentRefCallback} dangerouslySetInnerHTML={{ __html: prompt }} />
       )
 
-  const maybeInputForm = responseSubmitted
-    ? null
-    : (
+  const maybeInputForm = (
         <Formik
           initialValues={{ response: '' }}
           onSubmit={handleSubmit}
           validationSchema={schema}
         >
           {({ isSubmitting }) => (
             <Form>
-              <Field name="response" as="textarea" disabled={isSubmitting} rows={DEFAULT_TEXTAREA_ROWS} className="form-control my-3"/>
+              <Field name="response" as="textarea" disabled={isSubmitting || responseSubmitted} rows={DEFAULT_TEXTAREA_ROWS} className="form-control my-3"/>
               <ErrorMessage className="text-danger my-3" component="div" name="response" />
-              <button type="submit" disabled={isSubmitting} className="btn btn-outline-primary">{buttonText}</button>
+              <button type="submit" disabled={isSubmitting || responseSubmitted} className="btn btn-outline-primary">{buttonText}</button>
             </Form>
           )}
         </Formik>
-      )
+  )
 
   const maybeAck = !responseSubmitted
```

Comment:
Minor: Since the form is always rendered, as a minimal UI tweak what do you think about adding a bit of margin to the ack message so it's not up against the button? Something like:

```javascript
< div className="my-3" ref={contentRefCallback} dangerouslySetInnerHTML={{ __html: ack }} />
```